Sources/Extensions/Foundation/URLExtensions.swift

```diff
@@ -102,6 +102,23 @@ public extension URL {
         }
     }
 
+    /// SwifterSwift: Generates new URL that does not have scheme.
+    ///
+    ///        let url = URL(string: "https://domain.com")!
+    ///        print(url.droppedScheme()) // prints "domain.com"
+    public func droppedScheme() -> URL? {
+        if let scheme = self.scheme {
+            let droppedScheme = String(self.absoluteString.dropFirst(scheme.count + 3))
+            return URL(string: droppedScheme)
+        }
+
+        guard host != nil else {
```

Comment:
Would you mind moving the else block on the same line?

Reply:
Done